Dart/src/com/jetbrains/lang/dart/DartBundle.properties

```diff
@@ -98,10 +98,17 @@ fields.to.generate.setters=Select Fields to Generate Setters
 fields.to.generate.getters.setters=Select Fields to Generate Getters and Setters
 dart.override.method=Choose method to override
 dart.implement.method=Choose method to implement
+
 runner.command.line.configuration.name=Dart Command Line App
 runner.command.line.configuration.description=Dart Command Line Application
+
 remote.debug.configuration.name=Dart Remote Debug
 remote.debug.configuration.description=Debug Remote Dart Command Line Application
+
+webdev.debug.configuration.name=Dart Web
+webdev.debug.configuration.description=Debug Dart Webdev Application
```

Comment:
AFAIU, webdev is just a tool that helps to deal with Dart web applications. Does 'Webdev Application' make any sense?

Reply:
Good point, updated to `Debug Dart Webdev`, I was just trying to be consistent... `Debug using Dart Webdev` would also be fine.

Reply:
Or just, 'Debug Dart web application'?

Reply:
>Or just, 'Debug Dart web application'?

👍 That was my thought too.
Or 'Debug Dart web application using the webdev tool'